src/Stratis.Bitcoin/P2P/Peer/NetworkPeer.cs

```diff
@@ -261,13 +287,16 @@ public void StartReceiveMessages()
 
                     this.Peer.LastSeen = this.dateTimeProvider.GetUtcNow();
                     this.Peer.Counter.AddRead(message.MessageSize);
-                    this.Peer.OnMessageReceived(new IncomingMessage()
+
+                    var incommingMessage = new IncomingMessage()
```

Comment:
`incomingMessage` not `incommingMessage` ;)

Reply:
thanks, fixed